### DISCUSSION TITLE
Amasty Shopby - Attribute Search

### DISCUSSION THREAD

**Discussion by johndiesel123** [breezefront/community#304] [1. Help] [answered]
I was wondering if the attribute search should function when using the Amasty Shopby plugin. When on a category page with a long list of possible attribute choices, the default functionality allows for searching within that attribute. If I override a page to bypass the Breeze framework the search works. I did install the Breeze add-on plugin for Amasty Shopby.

![image](https://github.com/user-attachments/assets/76426946-b2ab-48ca-824a-704c1e6a6350)
![image](https://github.com/user-attachments/assets/be3dbc0e-62f9-4923-b2fd-37304b2b6558)

**Comment by roma84** [comment, marked as answer]
Please update the compatibility module: https://github.com/breezefront/module-breeze-amasty-shopby/commit/39ff4920e6dda67c05329cc972b05e9fad4b57a3

**Comment by johndiesel123** [comment]
Awesome!! I'll install this shortly! Thanks so much!

**Comment by johndiesel123** [comment]
Sorry for the delayed response. This resolved the issue. Thanks so much for the outstanding response time! Very impressive!
